frontend/packages/core/src/chip.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { ChipProps as MuiChipProps } from "@material-ui/core";
+import { Chip as MuiChip } from "@material-ui/core";
+
+export interface ChipProps extends Pick<MuiChipProps, "label"> {
+  variant: "WORST" | "BAD" | "NEUTRALBAD" | "NEUTRAL" | "NEUTRALGOOD" | "GOOD" | "BEST";
+}
+
+const CHIP_BACKGROUND_COLOR_MAP = {
+  WORST: "#F9EAE7", // red
+  BAD: "#FEF8E8", // orange
+  NEUTRALBAD: "#E2E2E6", // greyish
```

Comment:
I think we will need to set the border independently of the background. Right now this makes the border more  black than the designs have it
Storybook:
![Screen Shot 2021-04-21 at 1 20 07 PM](https://user-images.githubusercontent.com/1004789/115615838-520f4200-a2a4-11eb-95cf-cbbbbaced0ca.png)
Figma:
![Screen Shot 2021-04-21 at 1 20 13 PM](https://user-images.githubusercontent.com/1004789/115615865-576c8c80-a2a4-11eb-9e3e-d514605d38eb.png)

Reply:
Ah, I see there is an alpha difference.